lua/admin.lua

```diff
@@ -64,7 +64,7 @@ drive_on_right = {
 ["South Africa"] = "false",
 ["Sri Lanka"] = "false",
 ["Suriname"] = "false",
-["Scotland"] = "false",
+["Alba / Scotland"] = "false",
```

Comment:
doesnt this mean that someone converting an older planet will also get screwed? i think the compatible fix would be to include both variants right? just support both for a time. its a bit of baggage to carry but its better than getting bug reports :D